### PR TITLE
Remove iframe for Manage Torrents, replace it with a new tab link to …

### LINKED ISSUE #1779: Manage Torrents link missing when using HTTPS
Branch:Master
OS:Ubuntu 14.04
Python: 2.7.6

I setup the torrent search provider to use Deluge (via WebUI). After saving it the Manage Torrents link isn't in the menu. If I go to https://example.com/manage/manageTorrents/ Deluge loads in the iframe fine. If I disable SSL for deluge and go back into the torrent search provider settings and change https://example.com:8112 to http://example.com:8112 the Manage Torrents link appears in the menu again.

Logs:None

### LINKED ISSUE #3342: Subresource requests whose URLs contain embedded credentials soon deprecated
**Branch/Commit:** master 
**OS:** Ubuntu
**What you did:** Access to "Manage torrents"
**What happened:** Google chrome console show warning saying that a request will soon be deprecated 
**What you expected:** No warning

```
Subresource requests whose URLs contain embedded credentials (e.g. `https://user:pass@host/`) are
deprecated, and will be blocked in M59, around June 2017. See
https://www.chromestatus.com/feature/5669008342777856 for more details.
```
warning

### LINKED ISSUE #3500: Manage Torrents blank page in Chromium and Firefox
Branch/Commit:  Master / v2017.03.25-1
OS: Linux Mint 18.1 Cinnamon 64 bit
What you did: Navigated to Manage -> Manage Torrents
What happened: See blank page, not torrents from Qbittorrent as in the past
What you expected:  To see Qbittorrent torrents as I did in the past
Logs:  The error is in Chromium, if I do right click and INSPECT on the black Manage Torrents page I see an error and warning:

Refused to display 'http://admin:fakepword@sysname:8080/' in a frame because it set 'X-Frame-Options' to 'SAMEORIGIN'.
stickyHeaders" widget was loaded more than once!
addWidget @ vender.min.js?27457:30
rootDirs.js?27457:26 setting default to undefined

This stopped working a few weeks ago, I'm not sure exactly when.   I put the error/warning message from Chromium above, I tried loading the page in Firefox and also shows black in the box the torrents would have shown up historically.

### PR DESCRIPTION
**Pull request by miigotu**
…the torrent client webui. Also removes the autologin to the torrent webui

Closes #3500
Closes #3342
Closes #1779

Fixes #

Proposed changes in this pull request:
-
-
-

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by sharkykh** [comment]
Link seems to have disappeared completely here. Checking..

**Comment by miigotu** [comment]
The url has to be valid for it to show up

File: sickbeard/helpers.py [hunk 1]
```diff
@@ -1867,3 +1868,29 @@ def sortable_name(name):
...
+        else:
+            add_site_message(
+                '<p>' + _('For best results please set the Download Station alias as') + ' <code>download</code>. ' +
+                _('You can check this setting in the Synology DSM') + ' ' + '<b>' + _('Control Panel') + '</b> > <b>' + _('Application Portal') + '</b>.' +
+                _('Make sure you allow DSM to be embedded with iFrames too in') + ' ' + '<b>' +
+                _('Control Panel') + '</b> > <b>' + _('DSM Settings') + '</b> > <b>' + _('Security') + '</b>.' +
+                '</p><br>', 'info'
+            )
+
+    if check_url(torrent_ui_url):
```
Comment (miigotu) [inline sickbeard/helpers.py:1894]: because of this

**Comment by miigotu** [comment]
A similar link should probably be added for nzbget/sabnzbd

File: sickbeard/helpers.py (hunk 1 above)
Comment (sharkykh) [reply sickbeard/helpers.py:1894]: but my lan ip is `192.168.xy.abc` for example, my uTorrent url in the settings is `http://192.168.xy.abc:port/`... tried to dry-test with the functions but can't go into debugging right now - can you see an issue with the url that I can't?

Comment (miigotu) [reply sickbeard/helpers.py:1894]: is https enabled on your sickrage?

Comment (sharkykh) [reply sickbeard/helpers.py:1894]: nope, neither on uTorrent as you can see

Comment (sharkykh) [reply sickbeard/helpers.py:1894]: it's also the same machine, I just use my IP address

Comment (miigotu) [reply sickbeard/helpers.py:1894]: And can you access the utorrent gui on http://192.168.xy.abc:port/gui/ ?

Comment (sharkykh) [reply sickbeard/helpers.py:1894]: Yup

File: sickbeard/helpers.py [hunk 2]
```diff
@@ -1867,3 +1868,29 @@ def sortable_name(name):
     if not sickbeard.SORT_ARTICLE:
         name = re.sub(r'(?:The|A|An)\s', '', name, flags=re.I)
     return name.lower()
+
+
+def manage_torrents_url():
+    if not sickbeard.USE_TORRENTS or sickbeard.TORRENT_METHOD == 'blackhole' or \
+                    sickbeard.ENABLE_HTTPS and not sickbeard.TORRENT_HOST.lower().startswith('https'):
```
Comment (sharkykh) [inline sickbeard/helpers.py]: I think the cause is the `not` should encompass everything?

Comment (sharkykh) [inline sickbeard/helpers.py]: Or just `sickbeard.TORRENT_METHOD == 'blackhole'` should be `sickbeard.TORRENT_METHOD != 'blackhole'`

File: sickbeard/helpers.py (hunk 1 above)
Comment (miigotu) [reply sickbeard/helpers.py:1894]: I didnt really change how the code works, but I also did not do a live test for it

Comment (sharkykh) [reply sickbeard/helpers.py:1894]: Ok, I'm debugging but I'm on the wrong computer that might not be able to handle it ;)
not enough memory haha

Comment (miigotu) [reply sickbeard/helpers.py:1894]: Do you see an error in the log?
Mine says "04:31:54 INFO::Thread-15 :: Request failed: 401 Client Error: Unauthorized for url: http://192.168.10.10:9091/"

Comment (sharkykh) [reply sickbeard/helpers.py:1894]: Oops. Didn't check logs
It outputs `Request failed: 400 Client Error: ERROR for url: http://ip:port/gui`
like 10-15 times and stops

Comment (miigotu) [reply sickbeard/helpers.py:1894]: Yep thats why.....
Pull, it should work now

**Pull request merged**